docker-compose.example.sqlite.yml

```diff
@@ -1,20 +1,26 @@
-version: "3.7"
+version: "2.4"
 services:
   app:
     image: babybuddy/babybuddy
+    # Container_name is an optional value for single-container deployments to set a friendly container name.
+    # Disable in swarm or other multi-container setups.
+    container_name: babybuddy
     # See README.md#configuration for other environment configuration options.
     environment:
-      - ALLOWED_HOSTS=
+      - ALLOWED_HOSTS=localhost   # comma separated list of IP addresses or hosts that can access the web UI
       - DJANGO_SETTINGS_MODULE=babybuddy.settings.base
-      - SECRET_KEY=
-      - TIME_ZONE=
-      - DEBUG=False # Turn to False in production
+      - SECRET_KEY=               # Generate a random string here to secure the Django instance
+      - TIME_ZONE=                # In the tzdata format, IE, "America/Denver"
+      - DEBUG=False               # Turn to False in production
     volumes:
       - data:/app/data:rw
       - media:/app/media:rw
     command: bash -c 'python manage.py migrate --noinput && python manage.py createcachetable && gunicorn babybuddy.wsgi -b :8000 --log-level=info'
     ports:
       - "8000:8000"
+    # For Portainer deployment, select an alternate port for babybuddy, such as port 7000
+    # - "7000:8000"
+    restart: unless-stopped
```

Comment:
What does this setting do?

Reply:
`restart:` sets how the container should behave if it is stopped by another process, or if the container crashes. `unless-stopped` means the container will be automatically restarted if it stops for any reason, other than if the user stopped it intentionally. I've found it a common addition to compose files for services that you would want to restart if they crashed or the machine reboots. https://docs.docker.com/compose/compose-file/compose-file-v2/#restart